platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/MerkleRoot.java

```diff
@@ -32,6 +32,14 @@ public interface MerkleRoot extends MerkleInternal {
     @NonNull
     SwirldState getSwirldState();
 
+    /**
+     * This method makes sure that the platform state is initialized.
+     * If it's already initialized, it does nothing.
+     */
+    default void initPlatformState() {
```

Comment:
My personal preference would be to *not* provide the empty default implementation, and just have an empty implementation for impls that don't need to do anything.

Empty default methods can be bug prone, since any new impls might mistakenly miss the method.

Up to you though

Reply:
Sure, I removed default empty implementation